commands/verify/verify.go

```diff
@@ -107,6 +115,16 @@ func concurrentSafeErrAppend(concurrentError *concurrentErrors, err error) {
 	concurrentError.mutex.Unlock()
 }
 
+func concurrentSafeAdd(summary *summary, ok bool) {
+	summary.mutex.Lock()
+	if ok {
+		summary.successNum++
+	} else {
+		summary.errNum++
+	}
+	summary.mutex.Unlock()
```

Comment:
Use `defer` to unlock

```suggestion
	summary.mutex.Lock()
	defer summary.mutex.Unlock()
	if ok {
		summary.successNum++
	} else {
		summary.errNum++
	}
```

Reply:
OK！Thanks for your advice!